packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -62,7 +59,17 @@ import 'thumb_painter.dart';
 ///  * [Switch], the Material Design equivalent.
 ///  * <https://developer.apple.com/design/human-interface-guidelines/toggles/>
 class CupertinoSwitch extends StatefulWidget {
-  /// Creates an iOS-style switch.
+  /// Creates a iOS-style switch.
+  ///
+  /// The switch itself does not maintain any state. Instead, when the state of
```

Comment:
Is this true? It is a stateful widget, and it does have some widget state properties